README.md

```diff
@@ -15,7 +15,7 @@ function getBlockByHash(hash: string): Promise<Block|null> {
         body: { jsonrpc: "2.0", id: 1, method: "eth_getBlockByHash", params: [hash, false] }
     }).then(response => response.json());
 }
-//function getBlockByHashCallbackStyle(hash: string, callback: (block: Block|null) => void): void {
+//function getBlockByHashCallbackStyle(hash: string, callback: (error: Error, block?: Block) => void): void {
```

Comment:
Ah, good catch on this.  It should actually be `error?: Error, block?: Block` (both are optional).